model/resource/telemetry.yaml

```diff
@@ -25,28 +25,40 @@ groups:
           members:
             - id: cpp
               value: "cpp"
+              stability: experimental
```

Comment:
TODO: most of these values should be stable, checking.

Reply:
Updated based on the stability of the corresponding OpenTelemetry SDK.
All OTel SDKs set the attribute, and only rust is not stable yet, so leaving the corresponding constant as experimental

Reply:
I think you could stabilize rust here too